src/components/home/sections/Assessment.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+const Assessment = () => {
+  const { t } = useTranslation();
+  return (
+    <div className="assessment">
+      <div>
+        <h1>
+          {t("assessment.heading0")}
+          <br />
+          {t("assessment.heading1")}
+        </h1>
+        <p>{t("assessment.paragraph0")}</p>
+        <br />
+        <div>
+          <Link to="psychoAssess">
+            <Button id="blue" size="lg" variant="primary">
+              {t("assessment.button0")}
+            </Button>{" "}
+          </Link>
+        </div>
+      </div>
+
+      <div>
+        <img
+          src="https://files.slack.com/files-pri/TRG8KC72A-F017H9W00SV/assessment.png"
```

Comment:
change the image path to another link or upload the image and import it here. this link won't work on other people's  machines